packages/pipeline/migrations/1543434472116-TokenOrderbookSnapshots.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const tokenOrderbookSnapshots = new Table({
+    name: 'raw.token_orderbook_snapshots',
+    columns: [
+        { name: 'source', type: 'varchar', isPrimary: true },
+        { name: 'retrieval_timestamp', type: 'bigint', isPrimary: true },
```

Comment:
Can we use the same convention as in Alex's sra_orderbook table (i.e. call the retrieval_timestamp the observation_timestamp?)